### DISCUSSION TITLE
Julio Herrera | Is UVdesk Under Maintenance, No Signup Option?

### DISCUSSION THREAD

**Discussion by julioherreraa** [uvdesk/community-skeleton#759] [Q&A]
Hello my name is Julio Herrera. How can I sign up for a UVdesk account, as there is no option for signup? Only login is available. Is the UV desk under maintenance? GitHub login is also failing! Give suggestions.

**Comment by nicokr01** [comment]
https://uvdesk.com/en/helpdesk/register

**Comment by nicokr01** [comment]
Or did you mean the a Register option in the UVdesk Ticket System WebApp / APP ? (customer Accounts ?)
